Added docs for Conversion and Review Tools (#112)
openlawteam/docs: docs/conversion-tool/README.md

## HTML agreement conversion

There are two alternatives to use the conversion tool command line or embedded in a Scala/Java project, i.e:

**1. Command line**

```
$ java -jar openlaw-md-converter-x.y.z.jar \
 --inputDir "/path/to/htmls/" \
 --outputDir "/path/to/store/converted/files/" \
 --outputExt ".txt"
```

> It reads all the HTML files located at the 'inputDir' and writes the converted files to the 'outputDir' with the `.txt` extension.

**2. Embedded**

```scala
import java.io.File
import java.nio.file.Paths
import org.openlaw.app.ConversionOptions
import org.openlaw.agreement.provider.DefaultProvider
import scala.util.{Failure, Success, Try}

val htmlFile: File = Paths.get("/path/to/html/file").toFile
val outputDir = "/path/to/output/dir/"
val provider = LawInsiderProvider(ConversionOptions(outputDir))
Try(provider.convertAgreement(htmlFile)) match {
    case Success(markdown) => {
      markdown.printToFile(outputDir, ".txt")
    }
    case Failure(_) => {
      //log
    }
}
```

> It uses the `LawInsiderProvider` which has custom rules to curate the HTML files collected from their platform.

> The result of both calls are exact the same, a markdown file with `.txt` extension.

Reviewer: -> exactly the same

## Typed variable identification and conversion

It is important to mention that the conversion process handles not just the transformation
from HTML to Openlaw Markup Language but also it identifies and creates the Typed Variables, such as:

- Parties
- Address
- Signatory
- Date
- Number:
  - Currency, Phone Number (North America), Equity and Percentage.

Apart from `Parties` variables all the identification and creation is based on [Regular Expressions](#https://en.wikipedia.org/wiki/Regular_expression).
In order to identify the parties of a contract we use [our own Natural Language Processing Model](../natural-language-processing/#openlaw-natural-language-processing-model).

> The conversion tool is used internally to populate library of agreements.

Reviewer 2: ```suggestion
The conversion tool is used internally to populate a library of agreements.
```

> With +1000 collected agreements from lawinsider.com, converted to Openlaw Markup Language and uploaded to [Review Tool](../review-tool/#review-tool).

Reviewer 2: ```suggestion
With 1000+ collected agreements from LawInsider, converted to the OpenLaw Markup Language and uploaded to the [Review Tool](/review-tool/), the files are ready to be reviewed.
```

> The files are ready to be reviewed. Once the review phase is completed, the templates can be uploaded Openlaw.io public library.

Reviewer 2: ```suggestion
Once the review phase is completed, the templates can be uploaded to the openlaw.io public library.
```
